spi/src/main/java/org/jboss/pnc/spi/builddriver/BuildDriverStatus.java

```diff
@@ -25,7 +25,21 @@
  * Created by <a href="mailto:matejonnet@gmail.com">Matej Lazar</a> on 2014-11-23.
  */
 public enum BuildDriverStatus {
-    SUCCESS, FAILED, UNSTABLE, BUILDING, ABORTED, CANCELLED, UNKNOWN;
+    SUCCESS, FAILED(true), UNSTABLE, BUILDING, ABORTED(true), CANCELLED(true), UNKNOWN;
```

Comment:
Aborted and canceled doesn't have errors. Maybe we should rename field to successfullyCompleted.